vmm/src/vm.rs

```diff
@@ -694,6 +699,15 @@ impl Vm {
             // it will be running with the last configure memory size.
             self.config.lock().unwrap().memory.size = desired_memory;
         }
+
+        if let Some(desired_balloon) = desired_balloon {
+            self.memory_manager
+                .lock()
+                .unwrap()
+                .balloon_resize(desired_balloon << 20)
```

Comment:
`desired_balloon` should already be the size in bytes, so please don't shift it here.

Reply:
Fixed.